every_election/apps/elections/baker.py

```
@@ -0,0 +1,52 @@
+import logging
+import os
+from typing import Optional
+
+import boto3
+
+logger = logging.getLogger(__name__)
+
+
+CURRENT_ELECTIONS_PARQUET_EVENT_QUEUE_URL = os.environ.get(
+    "CURRENT_ELECTIONS_PARQUET_EVENT_QUEUE_URL"
+)
```

I think we should read this in via django settings instead of direct from the env here. Also, when we do that we should throw a `KeyError` error if `DC_ENVIRONMENT` is one of `["development", "staging", "production"]` and `CURRENT_ELECTIONS_PARQUET_EVENT_QUEUE_URL` is not set. Lets not silently pass it.

---

Agree we should error if `DC_ENVIRONMENT` is set and  `CURRENT_ELECTIONS_PARQUET_EVENT_QUEUE_URL` isn't. The reason I was getting it directly from the env, is because I assumed it would be stored in parameter store, the contents of which get written to the env.

---

It will be stored in parameter store, but as with other settings we read from the env, lets read then into django settings and then access them from `django.conf.settings` when we need them.

One benefit of this over accessing the env at runtime is we can perform all our consistency checks on the settings we read from the env at the point we bootstrap the app. Then if the settings are bad, we fail the deploy.